build.gradle.kts

```diff
@@ -54,14 +54,15 @@ subprojects {
     extra.set("versions", mapOf(
         // when updating these values, some values must also be updated in buildSrc as this map
         // cannot be accessed there
-        "opentelemetry" to "1.24.0",
-        "opentelemetry_semconv" to "1.24.0-alpha",
-        "opentelemetry_proto" to "0.11.0-alpha",
-        "opentelemetry_java_agent" to "1.24.0-alpha",
-        "opentelemetry_java_agent_all" to "1.24.0",
-        "opentelemetry_java_agent-tooling" to "1.24.0-alpha",
+        "opentelemetry" to "1.33.0",
+        "opentelemetry_semconv" to "1.21.0-alpha",
```

Comment:
semconv version went down?

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/v1.33.0/dependencyManagement/build.gradle.kts#L50

It changed from `io.opentelemetry:opentelemetry-semconv` to `io.opentelemetry.semconv:opentelemetry-semconv`